app/serializers/group_serializer.rb

```diff
@@ -13,6 +13,7 @@ class GroupSerializer < ActiveModel::Serializer
              :members_can_raise_proposals,
              :members_can_vote,
              :memberships_count,
+             :members_count,
```

Comment:
memberships_count is rubbish and should not be trusted. It's just a counter_cache: true which does not take into account suspended, archived.